tests/system/providers/amazon/aws/example_bedrock.py

```diff
@@ -56,6 +56,12 @@
 # the code snippets for docs, and we can manually run the full tests.
 SKIP_LONG_TASKS = environ.get("SKIP_LONG_SYSTEM_TEST_TASKS", default=True)
 
+# No-commitment Provisioned Throughput is currently restricted to external
+# customers only and will fail with a ServiceQuotaExceededException if run
+# on the AWS System Test stack.
+SKIP_RESTRICTED_TASKS = environ.get("SKIP_RESTRICTED_SYSTEM_TEST_TASKS", default=True)
```

Comment:
Nit: Maybe its better to include the specific task with this flag (e.g. `SKIP_PROVISION_THROUGHPUT` etc.) I doubt that there will be more than one task that is restricted in a system test, in which case it makes sense to give the flag a more specific name.

In the case that there are multiple restricted tasks, you'd still be better off with different flags because users might want to enable one task, but not others.

Reply:
Yeah, I could do that.   My thinking with the name on the SKIP_LONG was that we might eventually have more than one and could set the skip or not flag as an environment variable and run the fulllllll tests on some sort of longer interval or trigger them manually, and I kind of just followed that naming convention here.    I think you are right, I don't see multiple tests having this kind of issue, so the name doesn't need to be so generic.  I'll rename it.